scene/2d/cpu_particles_2d.cpp

```diff
@@ -556,6 +584,13 @@ void CPUParticles2D::set_seed(uint32_t p_seed) {
 	seed = p_seed;
 }
 
+#ifdef TOOLS_ENABLED
+void CPUParticles2D::set_show_gizmos(bool p_show_gizmos) {
+	show_gizmos = p_show_gizmos;
+	queue_redraw();
```

Comment:
Should this also check `Engine::get_singleton()->is_editor_hint()`?

Reply:
set_show_gizmos only gets called by the particles_editor_plugin, so it already only runs in editor. So I don't believe that's needed.